### PR TITLE
BSTOR-90 - Fixed PriceWidget flickering in FF

### PR DESCRIPTION
**Pull request by Pacane**

### CODE REVIEW

File: client/src/main/java/com/arcbees/beestore/client/application/widget/PriceView.java [hunk 1]
```diff
@@ -129,4 +143,11 @@ private void setShopViewVisible(boolean visible) {
             $(priceView).show();
         }
     }
+
+    private boolean isPricePanel(Element element) {
+        if (!element.hasClassName(res.style().productSection())) {
```
Comment (Chris-V) [inline client/src/main/java/com/arcbees/beestore/client/application/widget/PriceView.java]: Inline?

**Comment by Chris-V** [comment]
LGTM

File: client/src/main/java/com/arcbees/beestore/client/application/widget/PriceView.java (hunk 1 above)
Comment (Pacane) [inline client/src/main/java/com/arcbees/beestore/client/application/widget/PriceView.java]: I'm using this exact same condition twice, hence the extraction

Comment (Chris-V) [inline client/src/main/java/com/arcbees/beestore/client/application/widget/PriceView.java]: `return !element.hasClassName(res.style().productSection())` ;)

Comment (Pacane) [inline client/src/main/java/com/arcbees/beestore/client/application/widget/PriceView.java]: Ah ben oui toé!

**Pull request merged**